docs/source/author/translations.rst

```diff
@@ -1,4 +1,79 @@
 Translations
 ##############################################################################
 
-Currently translations are not supported.
+Getting Started
+==============================================================================
+
+It's really easy to start contributing translations using CS Field Guide
+
+1. Read this documentation page, especially the `Translation Notes`_ section.
+2. Create an account on `Crowdin <https://crowdin.com/join>`__.
+3. Visit the `CS Unplugged Crowdin page <https://crowdin.com/project/cs-field-guide>`__.
+4. Start translating!
+
+.. note::
+
+  The in-context translation has been removed temporarily as translation infrastructure is being updated.
+  We aim to `re-add this feature in a future update <https://github.com/uccser/arnold/issues/4>`__.
```

Comment:
If it's going to be linked to in the docs that issue might need a bit more of a description

Reply:
Fair point. Not quite sure what to put in the description?